app_feup/pubspec.yaml

```diff
@@ -7,7 +7,7 @@ description: A FEUP no teu bolso
 # Both the version and the builder number may be overridden in flutter
 # build by specifying --build-name and --build-number, respectively.
 # Read more about versioning at semver.org.
-version: 1.0.0+17
+version: 1.0.0+18
```

Comment:
Since we've release 1.1.0, perhaps this should be 1.1.0+1? cc @reeckset

Reply:
Yeah, makes sense.

Reply:
This should be 1.1.0 already in develop already. Something went wrong somewhere I guess. Not this PR's concern, but do bump it up to 1.1.0 if you can